packages/grid/_modules_/grid/utils/getGridLocalization.ts

```diff
@@ -1,3 +1,4 @@
+import * as locales from '@material-ui/core/locale';
```

Comment:
This breaks tree-shaking, we need to import each locale in each translation file

Reply:
I think it needs to be here, I can import only the ones we need but this is where the object is built.

Reply:
Simply do:

```jsx
import { bgBG }  from '@material-ui/core/locale';
```

in the `bgBG` locale file and so one for each locale;

Reply:
Updated